teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age, name, specialization)
+    super(age, name)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Your teacher class constructor is missing the `parent_permission` parameter, which should have a default value of `true`.  Your constructor method should look similar to this :point_down: 
  
  ```ruby
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
      super(age, name, parent_permission)
      @specialization = specialization
    end
  ```